.changeset/small-rules-vanish.md

```diff
@@ -0,0 +1,6 @@
+---
+"@redocly/openapi-core": patch
+"@redocly/cli": patch
+---
+
+Fixed false positive errors when validating media type examples. Now, the linter won't error on valid examples that contain references.
```

Comment:
```suggestion
Fixed false positive errors with valid media type examples that contain references.
```
I think we should avoid "Now," in the changelogs and docs in general. Now, what do you think about that? 😆

Reply:
Let's have Roman's review on the solution firstly. It might turn out we don't need any changeset at all 🤣

Reply:
I checked it. LGTM.

Reply:
I'm sorry, I did already edit this changeset once, and while I agree that it's better without the "Now", I think it lost some context. Combining the two versions, how about this?
```suggestion
Fixed a problem where the linter incorrectly returned an error for valid examples that contain references.
```

Reply:
I'm sure pretty sure he is good to 😄

Go ahead and accept the suggestion + merge.